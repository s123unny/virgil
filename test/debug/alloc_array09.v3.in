b ../core/alloc_array09.v3 3
run
bt
info l
c
q
