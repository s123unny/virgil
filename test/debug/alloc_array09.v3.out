breakpoint|../core/alloc_array09.v3 3|0
stopOnBreakpoint
bt|main|../core/alloc_array09.v3|3
end
result|true
